feature-libs/product/multi-dimensional/components/multi-dimensional-container/multi-dimensional-selector/product-multi-dimensional-selector.component.ts

```diff
@@ -61,7 +68,50 @@ export class ProductMultiDimensionalSelectorComponent implements OnChanges {
     }
   }
 
-  isSelected(code: string) {
+  getSelectedValue(categoryName: string): string | null {
+    const category = this.categories.find((cat) => cat.name === categoryName);
+
+    if (category && this.selectedVariant) {
+      const selectedOption = category.variantOptions.find(
+        (option) => option.code === this.selectedVariant
+      );
+      return selectedOption ? selectedOption.value : '';
+    }
+
+    return null;
```

Comment:
I believe we should avoid using null values. It would be better to return an empty string instead.